packages/landingpage/project.json

```diff
@@ -6,28 +6,36 @@
   "tags": [],
   "targets": {
     "build": {
-      "outputs": ["{projectRoot}/out"],
-      "executor": "nx:run-commands",
+      "executor": "@nx/next:build",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "prod",
       "options": {
-        "cwd": "packages/landingpage",
-        "commands": ["next build"],
-        "parallel": false,
-        "color": true
+        "outputPath": "{projectRoot}/out",
+        "nextConfig": "{projectRoot}/next.config.mjs"
+      },
+      "configurations": {
+        "dev": {
+          "debug": true
+        },
+        "prod": {}
       }
     },
     "start": {
+      "executor": "@nx/next:server",
+      "defaultConfiguration": "dev",
+      "options": {
+        "buildTarget": "landingpage:build",
+        "dev": true
+      }
+    },
+    "preview": {
       "executor": "nx:run-commands",
       "options": {
-        "cwd": "packages/landingpage"
-      },
-      "defaultConfiguration": "dev",
-      "configurations": {
-        "dev": {
-          "command": "next dev -p 4600"
-        },
-        "prod": {
-          "command": "next start -p 4600"
-        }
+        "parallel": false,
+        "commands": [
+          "nx run landingpage:build",
+          "serve {projectRoot}/out"
+        ]
       }
     },
```

Comment:
We are doing the same now in the storybook (building the static files & serving them). See here:
https://github.com/inovex/elements/blob/c04dbc0dba494b09bfb8591919b2fd0e4a4c9464/packages/storybook/project.json#L42-L48

Might be better to do it like this because there is build-in support from NX.